nix/pkgs/plutus-haddock-combined/default.nix

```diff
@@ -6,6 +6,20 @@ haddock-combine {
   hspkgs = builtins.attrValues toHaddock;
   prologue = writeTextFile {
     name = "prologue";
-    text = "Combined documentation for all the public Plutus libraries.";
+    text = ''
+      = Combined documentation for all the public Plutus libraries
```

Comment:
If this is getting a bit bigger maybe we should put it in a file somewhere? Fine for now, though.

Reply:
The description says it is documentation for "all public Plutus libraries" but it actually includes all documentation for all(?) dependencies which also makes it so huge.

Since the code is being changed now, can we adjust it so that it really _does_ only contain docs for the plutus libraries? I haven't looked at the implementation more closely i must admit, but i hope it is possible to somehow differentiate local packages from other ones - @michaelpj, you would probably know?

Reply:
I don't know how easy it is to change. This stuff is all depressingly non-standard: we had to write our own script to stitch together all the Haddock.

I guess ideally we'd get it to hyperlink to Hackage for external dependencies, but I don't know if that's possible...

Reply:
i see, i had no idea ;-(

Reply:
Pray for https://github.com/haskell/haddock/pull/1277!